conda-store-server/conda_store_server/_internal/server/views/ui.py

```diff
@@ -265,7 +265,11 @@ async def ui_get_user(
             entity, api.list_namespaces(db, show_soft_deleted=False)
         )
 
-        system_metrics = api.get_system_metrics(db)
+        system_metrics = api.get_system_metrics(db) or {
+            "disk_total": 0,
+            "disk_free": 0,
+            "disk_usage": 0,
+        }
```

Comment:
I think all the  `system_metrics` being 0 is still a valid response from `api.get_system_metrics`. So, I think this is not a good default value here. Instead, I think it would be better to let `system_metrics` be `None`, and check for that in the template. 


```suggestion
        system_metrics = api.get_system_metrics(db) 
```